src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
I was thinking about how you couldn't have `<xsl:if>` inside of the attribute set, but hadn't processed that it could be within the `<xsl:attribute>` element inside of the attribute set. Must be tired.

The only concern I have here with that approach is that this means removing the reference to attribute set `page-sequence.frontmatter`, which could have been used to override numbering for the entire front matter. You would now also need to override `page-sequence.notices` on its own. Unless the logic here moved into `page-sequence.frontmatter`, which just feels ... off ... that we would be checking for a back matter ancestor within the front matter attribute set:
```
  <xsl:attribute-set name="page-sequence.notice" use-attribute-sets="__force__page__count">
    <xsl:attribute name="format">
      <xsl:choose>
        <xsl:when test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">1</xsl:when>
        <xsl:otherwise>i</xsl:otherwise>
      </xsl:choose>
    </xsl:attribute>
  </xsl:attribute-set>
```